Caboodle/Sms/Sms.shared.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Sms
+    {
+    }
+
+    public class SmsMessage
+    {
+        public SmsMessage()
+        {
+        }
+
+        public SmsMessage(string body, string recipient)
```

Comment:
Should there be one with just the body? also what is a recipient? phone number?

Reply:
we could have overloads for each combo, or we could just let the devs use the properties.
The recipient... each platform just takes a string, so we could leave it as is and hope for the best :)